pymatgen/io/vasp/outputs.py

```diff
@@ -786,7 +786,7 @@ def run_type(self) -> str:
         if self.parameters.get("LUSE_VDW", False):
             run_type += "+rVV10"
         elif self.incar.get("IVDW") in IVDW_TYPES:
-            run_type += f"+vdW-{IVDW_TYPES[self.incar.get("IVDW")]}"
```

Comment:
I'm not quite sure why [this f-string failed in windows](https://github.com/materialsproject/pymatgen/actions/runs/8959057873/job/24604009920)?

```python
E     File "D:\a\pymatgen\pymatgen\pymatgen\io\vasp\outputs.py", line 789
E       run_type += f"+vdW-{IVDW_TYPES[self.incar.get("IVDW")]}"
E                                                      ^
E   SyntaxError: f-string: unmatched '('
```